app/assets/javascripts/account/templates/account-settings.html.slim

```diff
@@ -52,15 +52,15 @@
                   | {{'SIGN_IN.FORGOT_PASSWORD' | translate}}
               label.label-secondary for="auth_password"
                 | {{'LABEL.NEW_PASSWORD' | translate}}
-              input#auth_password ng-model="userAuth.user.password" name="password" placeholder="{{'CREATE_ACCOUNT.PASSWORD_SHORT_INSTRUCTIONS' | translate}}" type="password" ng-required="true" ng-class="{ error: inputInvalid('password'), 'margin-bottom': !inputInvalid('password') }" ng-pattern="passwordRegex" aria-describedby="auth_password_error"
+              input#auth_password ng-model="userAuth.user.password" name="password" placeholder="{{'CREATE_ACCOUNT.PASSWORD_SHORT_INSTRUCTIONS' | translate}}" type="password" ng-required="true" ng-class="{ error: inputInvalid('password'), 'margin-bottom': !inputInvalid('password') }" ng-pattern="passwordRegex" aria-describedby="auth_password_error" ng-model-options="{ allowInvalid: true }"
```

Comment:
awesome! I'll update on other pages where we use password reset so the messaging improves there too